doc/mgr/cli_api.rst

```diff
@@ -1,39 +1,55 @@
 CLI API Commands Module
 =======================
 
-The CLI API module exposes most ceph-mgr python API via CLI. Furthermore, this API can be
-benchmarked for further testing.
+The CLI API module exposes most of the ceph-mgr Python API via CLI.  This API
+can be benchmarked for further testing.
 
 Enabling
 --------
 
-The *cli api commands* module is enabled with::
+Enable the ``cli api commands`` module by running the following command:
 
-  ceph mgr module enable cli_api
+.. prompt:: bash #
 
-To check that it is enabled, run::
+   ceph mgr module enable cli_api
 
-  ceph mgr module ls | grep cli_api
+Ensure that the ``cli api commands`` module is enabled by running the following
```

Comment:
If it's called `cli_api` we should refer to it by that name here.  Alternately "Ensure that the CLI API commands module is enabled.."